tests/acceptance/WooCommerceSettingsTabCest.php

```diff
@@ -62,4 +67,20 @@ function checkWooCommercePluginSettingsAreDisabled(\AcceptanceTester $I) {
     $I->amOnPage("/wp-admin/admin.php?page=wc-settings&tab=email&section=wc_email_new_order");
     $I->dontSeeElementInDOM(self::CUSTOMIZE_SELECTOR);
   }
+
+  function showNoticeWhenWooCommerceCustomizerIsDisabled(\AcceptanceTester $I) {
+    $this->features->withFeatureEnabled(FeaturesController::WC_TRANSACTIONAL_EMAILS_CUSTOMIZER);
+    $this->transactional_emails->init(); // TODO: remove when WC_TRANSACTIONAL_EMAILS_CUSTOMIZER flag is removed
```

Comment:
Instead of this "hack", could you please try to `$I->deactivatePlugin('mailpoet');` and then reactivate it to trigger the populator? It will be slow but you can add the comment about removing it when the feature flag is removed.

This way of using `_inject` on some internal project class is something that's not really "allowed" and it's only due to old static models that it actually works in this case. Let's not add such things to our tests.